### PR TITLE
Add option to choose a specific make executable

### PR DESCRIPTION
**Pull request by paulfariello**
##### SUMMARY

Add option to choose a specific make executable in the make module.

Fix ansible-collections#211

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

Make module

### CODE REVIEW

**Comment by felixfontein** [comment]
You need to add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to) (`minor_changes`). (This would be a new feature, not a bugfix, but that's required since you add a new option.)

File: plugins/modules/system/make.py [hunk 1]
```diff
@@ -39,6 +39,10 @@
     description:
       - Use a custom Makefile.
     type: path
+  make:
+    description:
+      - Use a specific make binary
```
Comment (felixfontein) [inline plugins/modules/system/make.py:44]:
```suggestion
      - Use a specific make binary.
```

**Comment by mator** [comment]
shipit

**Comment by felixfontein** [comment]
shipit

(I guess now the bot should automerge.)

**Comment by felixfontein** [comment]
bot_status

**Comment by felixfontein** [comment]
@gundalow can you investigate why @mator's maintainer `shipit` wasn't counted by the bot?

**Comment by gundalow** [comment]
@felixfontein i believe its because of the changelog

**Comment by felixfontein** [comment]
@gundalow maybe because changelogs are classified as "core"? I guess we should fix that, since it means that no PR with a changelog can get automerged without our interaction... :)

**Comment by gundalow** [comment]
@paulfariello Thank you for this fix
@mator @felixfontein Thank you for your reviews.

**Pull request merged**